packages/frames.js/src/types.ts

```diff
@@ -225,6 +225,7 @@ export type FrameActionDataParsed = {
   };
   inputText?: string;
   state?: string;
+  transactionId?: `0x${string}`;
```

Comment:
we use `0x${string}` multiple times already, could we extract it to its own type and use it?

Reply:
I've been avoiding refactoring it into a type because it's simple enough to not make a mistake typing it out while being verbose enough to be immediately helpful. When you start calling it `Hex` or something you lose some of that clarity. Open to being convinced though

Reply:
It makes sense either way so I'm not strict about this.